bzt/modules/console.py

```diff
@@ -700,6 +703,26 @@ def add_data(self, data):
         self.body.append(Text(("stat-txt", "~Receive: %.3f" % recv),
                               align=RIGHT))
 
+class TestDuration(ListBox):
```

Comment:
Wow... "Cumulative Stats Header" means just text line that already exists, not new widget. Right now we have "Cumulative Stats" there, need to have "Cumulative Stats after 00:05:45"